modules/persistent-storage-csi-dynamic-provisioning-aws-efs.adoc

```diff
@@ -10,7 +10,7 @@
 
 [role="_abstract"]
 The AWS EFS CSI Driver supports a different form of dynamic provisioning than other CSI drivers. It provisions new PVs as subdirectories of a pre-existing EFS volume. The PVs are independent of each other. However, they all share the same EFS volume. When the volume is deleted, all PVs provisioned out of it are deleted too.
-The EFS CSI driver creates an AWS Access Point for each such subdirectory. Due to AWS AccessPoint limits, you can only dynamically provision 120 PVs from a single `StorageClass`/EFS volume.
+The EFS CSI driver creates an AWS Access Point for each such subdirectory. Due to AWS AccessPoint limits, you can only dynamically provision 1000 PVs from a single `StorageClass`/EFS volume.
```

Comment:
Given we bumped it to 1000, I think we can change the wording from 
"you can only dynamically provision 1000 PVs from a single `StorageClass`/EFS volume."
to
"you can dynamically provision up to 1000 PVs from a single `StorageClass`/EFS volume."